jest/moduleNameMapper.js

```diff
@@ -11,7 +11,7 @@ const {
 /**
  * Set a default `src` module alias for backward compatibility.
  *
- * TODO: drop this default in skuba v4.
+ * TODO: drop this default in skuba v5.
```

Comment:
I want to limit `v4` to the Node.js 14 upgrade to ease adoption. We can investigate other breaking changes in the next major.